Catalyst/source/index.rst

```diff
@@ -28,11 +28,11 @@ The main documentation for the site is organized into a couple sections:
 
    gettingstarted
    menu-options
-   settings
+   workflow
    design
    workzone
    trackinwrk

```

Comment:
Should this be "track" rather than "trackinwrk"

Reply:
All files and doctress  are with that name only.If i rename that i need to change lot of settings and data.In catalyst usermanual  it shows as "track" only.

Reply:
![image](https://user-images.githubusercontent.com/59429467/86216959-66ab0e00-bb9c-11ea-80c0-757ce7344144.png)